Tests/LibC/TestFenv.cpp

```diff
@@ -4,14 +4,17 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <AK/StdLibExtras.h>
 #include <LibTest/TestCase.h>
 #include <fenv.h>
 
 // TODO: Add tests for floating-point exception management.
 
-NEVER_INLINE float black_box(float value)
+static NEVER_INLINE float black_box_add(float a, float b)
```

Comment:
static should have been added to this function in the first commit that added the file, no?

..although before the taint for optimizer, that "black box" function wasn't very black boxy...

Reply:
With the addition of the taint_for_optimizer black box add function, do we still need to -O0 this file?